portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Blog - Anusha Datta</title>
+
+    <!-- Favicon, CSS & JS -->
+    <link rel="icon" href="images/AD.png" type="image/png" sizes="16x16">
+    <link rel="stylesheet" href="styles/common.css">
+    <link rel="stylesheet" href="styles/blog.css">
+    <script src="script.js"></script>
+
+    <!-- Font Awesome icons CDN -->
+    <script src="https://use.fontawesome.com/4e51920be5.js"></script>
+</head>
+
+<body>
+
+    <!-- Heading -->
+    <header id="home" class="showcase">
+        <div class="content">
+
+            <img src="images/AD.png" class="logo" alt="AD">
+            <h1 class="title">
+                BLOG
+            </h1>
+
+        </div>
+    </header>
+
+    <!-- List of Posts -->
+    <section class="bg-light">
+        <div class="container center">
+            <h1>COMING SOON</h1>
+            <p>Stay tuned for some exciting content!</p>
+        </div>
+
+        <hr class="center">
+    </section>
+
+    <!-- Contact -->
+    <section class="bg-light" id="contact">
+        <div class="container center">
+
+            <a href="mailto:anushadatta@gmail.com"><i class="fa fa-envelope"></i></a>
+
+            <a href="https://www.linkedin.com/in/anusha-datta/" target="_blank"><i class="fa fa-linkedin"></i></a>
+
+            <a href="https://github.com/anushadatta" target="_blank"><i class="fa fa-github"></i></a>
+
+            <a href="https://medium.com/@anushadatta" target="_blank"><i class="fa fa-medium"></i></a>
+
+        </div>
+    </section>
+
+    <!-- Footer -->
+    <footer class="center bg-dark">
+        <p>Anusha Datta &copy; 2020</p>
+    </footer>
+
+</body>
+
+</html>
```

Comment:
It's suggested to have newline at the end of file. Notice the red error icon on your last line.

Reply:
I was absolutely unaware of this, so glad it came up!!! Just researched it in more detail, and can now appreciate the rationale behind this practice. 👍 

I shall fix this in all files necessary.